waggle/plugin.py

```diff
@@ -57,11 +57,12 @@ def time_ns():
 
 
 class Message(NamedTuple):
+    vers: str
     name: str
-    value: Any
-    timestamp: int
-    src: str = ''
-    dst: str = ''
+    val: Any
+    ts: int
+    enc: str
```

Comment:
Similar comment for enc. enc is really just a way to track the data type when we serialize it. val will already have the python message type, so no need to track it as its own field.